src/styles/styles.js

```diff
@@ -361,6 +361,18 @@ const styles = {
         backgroundColor: themeColors.offline,
     },
 
+    floatingActionButton: {
+        backgroundColor: colors.green,
+        position: 'absolute',
+        height: 40,
+        width: 40,
```

Comment:
Same comment as above - use a variable for this.

Reply:
Done!